{{ cookiecutter.package_name }}/pyproject.toml

```
@@ -36,7 +36,8 @@ open_astronomy_package_template_example = "{{ cookiecutter.module_name }}.exampl
 test = [
     "pytest",
     "pytest-doctestplus",
-    "pytest-cov"
+    "pytest-cov",
+    "setuptools_scm",
```

I don't think we need to add this after #83.

```suggestion
    "pytest-cov",
```